### DISCUSSION TITLE
如果将每个机器人单独的真值轨迹在evo中显示成一条轨迹

### DISCUSSION THREAD

**Discussion by diamonazreal** [DapengFeng/S3E#25] [Q&A] [answered]
如果将每个机器人单独的真值轨迹在evo中显示成一条轨迹，--ref只能使用一个文件，如果直接显示就会有3条轨迹

**Comment by DapengFeng** [comment, marked as answer]
我猜您想表达的是如何xxx，我们是在evo的基础上实现了这个功能，思路是读取三个路径文件，然后用matplotlib给三个轨迹用不同颜色进行渲染。
